### PR TITLE
fix(prettier): use same `printWidth` as `max-len`

### PR DESCRIPTION
**Pull request by iam4x**
eslint & prettier were not enforcing the same line length, now they have the same line limit to 120 characters.

### CODE REVIEW

**Comment by Haroenv** [comment]
Shouldn’t we just use the 80 printwidth, I find that prettier makes the lines really long if you have a high print width like 120

**Comment by iam4x** [comment]
I agree @Haroenv maybe we can have 100, so it's in between. (And also I'm used to 100 characters on my projects, but this is personal preference).

I'll update the tests when we agree on some value.

**Comment by Haroenv** [comment]
100 works for me, I usually keep it 80, but seems like an okay middle ground. 80 is easy for side-by-side comparisons of files IMO

**Comment by vvo** [comment]
Yo there's something I do not get, our default setup uses eslint-plugin-prettier which will disable max-len rule from prettier: https://github.com/prettier/eslint-config-prettier/blob/e137ae7d6691bb0d7995275c0f1ecfef2f10cca0/index.js

I tried many setups and using only eslint + eslint-plugin-prettier is the best because prettier is not today capable to be active only on project where it's setup. So it will reformat project that have no eslint no prettier setup (which I do not want).

When did you experiment that max-len/prettier len issue?

**Comment by iam4x** [comment]
@vvo No issue, just to normalize rules from `eslint-config-algolia`. Here https://github.com/algolia/eslint-config-algolia/blob/master/rules.js#L164-L172 `max-len` is defined at 120 that's why I open the discussion.

**Comment by vvo** [comment]
Ok so that's just for being explicit. Can you fix the build? Should just be a lint --fix command to launch ./node_modules/.bin/eslint . --fix in sample project

or just manual fix

not sure why it failed.

**Comment by iam4x** [comment]
We have to agree on a value:

- prettier enforce max-len of 80
- eslint legacy rules enforced a max-len of 120

With @Haroenv we agreed to go in between for 100, what are your thoughts @vvo ?

**Comment by Haroenv** [comment]
80 or 100 wfm 😄

**Comment by vvo** [comment]
This would require reformat projects and I am unsure about the benefits. Can we keep 120?

I moved away from 80 because the 80 line limit reason was a legacy one where you could then edit lines on old terminal from a non graphical OS easily. Well nowadays we are working on screens that are so big we use them to watch movies so we should be fine?

**Comment by vvo** [comment]
If it really makes you slower or you struggle day to day with 120 then please add more comments, otherwise let's merge this.

**Comment by iam4x** [comment]
I'm fine with it, tests should pass now 👍

File: rules.js [hunk 1]
```diff
@@ -263,6 +263,6 @@ module.exports = {
     'import/no-amd': ['error'],
     'import/no-commonjs': ['error'],
     'import/no-extraneous-dependencies': ['error'],
-    'prettier/prettier': ['error', { trailingComma: 'es5', singleQuote: true }],
+    'prettier/prettier': ['error', { trailingComma: 'es5', singleQuote: true, printWidth: 120 }],
```
Comment (Haroenv) [inline rules.js]: It's strongly discouraged to move it to 120 in the prettier repo. 80 is a lot more readable

Comment (vvo) [inline rules.js]: ref: https://github.com/prettier/prettier/commit/801519ec79ecaa79e012226ff6b888ea75ba7449

good to know indeed. I am inclined to try 80 and see how it goes.

**Comment by vvo** [comment]
Can you move to 80 and update README too (the part where we reformat everything).

This change is a breaking one and we should then reformat projects (no big deal)

**Comment by iam4x** [comment]
In fact it's not a breaking change if we revert to 80 in this PR, because prettier > eslint when enforcing code style so it will behave as the same as before.

Pushing the `max-len: 80` 👍

**Pull request merged**